pages/submission/SubmissionHandler.inc.php

```diff
@@ -54,12 +54,13 @@ function fetchChoices($args, $request) {
 	 * Get the step numbers and their corresponding title locale keys.
 	 * @return array
 	 */
-	protected function _getStepsNumberAndLocaleKeys() {
+	function getStepsNumberAndLocaleKeys() {
 		return array(
 			1 => 'submission.submit.prepare',
 			2 => 'submission.submit.upload',
 			3 => 'submission.submit.catalog',
-			4 => 'submission.submit.nextSteps'
+			4 => 'submission.submit.confirmation',
+			5 => 'submission.submit.nextSteps',
```

Comment:
Currently the locale keys for OMP and OJS differ here. I've opted not to reconcile that yet; will have a look when I port over to OJS.